### PR TITLE
add Flowers notebook

### PR DESCRIPTION
**Pull request by RatanRSur**

### CODE REVIEW

File: notebooks/samples/305 - Flowers ImageFeaturizer.ipynb [hunk 1]
```diff
@@ -0,0 +1,458 @@
...
+    "dataURL = cdnURL + \"/Flowers/\" + dataZipFile\n",
+    "localDataDir = \"/tmp/Flowers/\"\n",
+    "localDataFile = join(localDataDir, dataFile)\n",
+    "localDataZipFile = join(localDataDir, dataZipFile)\n",
+    "\n",
+    "import subprocess\n",
+    "if subprocess.call([\"hdfs\", \"dfs\", \"-test\", \"-d\", localDataDir]):\n",
+    "    from urllib import urlretrieve\n",
+    "    print subprocess.check_output(\n",
+    "          \"rm -rf {0} && mkdir -p {0}\".format(localDataDir),\n",
```
Comment (RatanRSur) [inline notebooks/samples/305 - Flowers ImageFeaturizer.ipynb]: delete after distribution to hdfs

File: notebooks/samples/305 - Flowers ImageFeaturizer.ipynb [hunk 2]
```diff
@@ -0,0 +1,461 @@
...
+   "metadata": {},
+   "source": [
+    "### Plot confusion matrix."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "aml-deploy": "hdinsight",
```
Comment (drdarshan) [inline notebooks/samples/305 - Flowers ImageFeaturizer.ipynb]: This cell should work on both HDI and local - could you please remove the metadata tag and check?

File: notebooks/samples/305 - Flowers ImageFeaturizer.ipynb [hunk 3]
```diff
@@ -0,0 +1,461 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from mmlspark import ImageReader, ImageFeaturizer, UnrollImage, ImageTransformer, TrainClassifier, \\\n",
```
Comment (drdarshan) [inline notebooks/samples/305 - Flowers ImageFeaturizer.ipynb]: Line too long?

Comment (drdarshan) [reply notebooks/samples/305 - Flowers ImageFeaturizer.ipynb]: And maybe add mmlspark imports after pyspark?

File: notebooks/samples/305 - Flowers ImageFeaturizer.ipynb [hunk 4]
```diff
@@ -0,0 +1,461 @@
...
+   "source": [
+    "dataFile = \"flowers_and_labels.parquet\"\n",
+    "dataZipFile = dataFile + \".zip\"\n",
+    "cdnURL = \"https://mmlspark.azureedge.net/datasets\"\n",
+    "dataURL = cdnURL + \"/Flowers/\" + dataZipFile\n",
+    "localDataDir = \"/tmp/Flowers/\"\n",
+    "localDataFile = join(localDataDir, dataFile)\n",
+    "localDataZipFile = join(localDataDir, dataZipFile)\n",
+    "modelName = \"ResNet50\"\n",
+    "modelDir = \"file:\" + abspath(\"models\")\n",
```
Comment (drdarshan) [inline notebooks/samples/305 - Flowers ImageFeaturizer.ipynb]: Please add a new line before the ``def`` below.

**Pull request merged**